src/game/common/modules/behaviormodule.h

```diff
@@ -116,18 +116,20 @@ class BehaviorModule : public ObjectModule, public BehaviorModuleInterface
     IMPLEMENT_ABSTRACT_POOL(BehaviorModule)
 
 protected:
+    BehaviorModule(Thing *thing, const ModuleData *module_data);
     virtual ~BehaviorModule() override;
 
 public:
-    BehaviorModule(Thing *thing, ModuleData *module_data);
-
     virtual StealthUpdate *Get_Steath();
     virtual SpyVisionUpdate *Get_Spy_Vision_Update();
 
+    // Snapshot
     virtual void CRC_Snapshot(Xfer *xfer) override;
     virtual void Xfer_Snapshot(Xfer *xfer) override;
     virtual void Load_Post_Process() override;
+    //~Snapshot
```

Comment:
as ~ClassName implies constructor I would advise against using this to mark areas. Just use `#pragma region Snapshot` then at least it can be collapsed.

Reply:
~ just marks the end here as a comment. No need to overcomplicate I think.

Reply:
It goes against the style of the rest of the codebase and can cause confusion.